cc/mcts_player.cc

```diff
@@ -407,4 +407,25 @@ void MctsPlayer::ProcessLeaves(absl::Span<MctsNode*> leaves,
   }
 }
 
+void FindBleakestMove(const MctsPlayer& player, int* move, float* q) {
```

Comment:
This should probably return a bool indicating whether `move` and `q` were updated, depending on whether `player.history()` is empty.

I see it's currently fine because `FindBleakestMove` isn't called if the history is empty, but we might end up reusing this function in the future.

Reply:
Makes sense. Will do.